index.md

```diff
@@ -136,8 +165,25 @@ Regarding metadata, [Elizabeth Wickes][wickes-comment-metadata] writes:
     *   Use configuration files or command-line arguments for parameters
 8.  Submit code to a reputable DOI-issuing repository just like data
 
+### Discussion
+
+FIXME
+
 ## Collaboration
 
+### Goals
+
+1.  Again, the easier it is for people to collaborate, the more likely they are to do so
+    (and to give you credit)
+2.  Two most reported barriers from [Steinmacher et al][steinmacher-newcomers]
+    are finding a task to start on
+    and problems setting up the local workspace to start work
+    *   So remove those
+3.  Additional disincentive is uncertainty: what am I allowed to do?
+    *   So remove that as well
```

Comment:
Maybe add that having no license basically says "I am keeping all copyright and you are not allowed to do anything"?